contracts/domainRegister.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+contract DomainRegister {
+    struct DomainInfo {
+        address controller;
+        uint256 registrationTime;
+    }
+    uint256 public fee;
+    address public owner;
+    uint public totalDomains;
+    mapping(string => DomainInfo) public domains;
```

Comment:
line 12 - 13 
as good practice? you do not need to store same data in two different places
// - 1 point